src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
`justify-content: space-between;`
There is no specific heigh, so this rule does nothing.